### ISSUE TITLE
Catch-Up Meeting 12 December 2024

### ISSUE THREAD

**Issue by chelseagraham** [NHMDenmark/Herbarium-Sheets-workstation#137]
Need feedback from anyone who digitises using the DigiApp: https://github.com/NHMDenmark/Mass-Digitizer/issues/526

Also check that CMs are happy with this change: https://github.com/NHMDenmark/Herbarium-Sheets-workstation/issues/160

**Comment by RebekkaML** [comment]
As far as I know #160 was already discussed at the last Herbarium meeting (14.11.2024, ticket #136 ) and they agreed that this makes sense.

**Comment by RebekkaML** [comment]
Another update: for rebarcoding (#153 ) I reprogrammed the barcode Scanner at Herb01 so that it enters a "tab" key after each scan. For the DigiApp, however, we need it to do an "enter" key after scanning. I printed out the necessary barcodes for programming the scanner and placed them on the barcoding table at Herb01, so if the CMs want to digitize, they can just scan the code to switch the scanner back to the DigiApp mode.
